### ISSUE TITLE
error 017: undefined symbol "var"; did you mean "var"?

### ISSUE THREAD

**Issue by On3d4y** [pawn-lang/compiler#405]
### Issue description:

var in wrong state is wrongly found

### Minimal complete verifiable example (MCVE):

```pawn
new var <state_a>;

main()
{
	var = 5;
}
```

### Workspace Information:

* Compiler version: 3.10.9
* Command line arguments provided (or sampctl version): pawno
* Operating System: Windows 8

**Comment by Daniel-Cortez** [comment]
I'll look into it, but I'm unsure what is supposed to be the correct output in such situations.
Should we suggest "var<state_a>" for the above example (which, IMHO, would be kind of misleading, because if the user types that instead of just "var", it will only cause a bunch of new errors), or rather don't suggest anything?
@Y-Less @YashasSamaga ?

**Comment by Y-Less** [comment]
Maybe something like `undefined symbol "var"; state var not in scope`?

**Comment by Daniel-Cortez** [comment]
Is the code in the example even valid? AFAIK, in Pawn only functions may have states, not variables.
Even if we do something similar with a function instead of a variable
```Pawn
func() <automaton_1:STATE_1> {}

main()
{
	func();
}
```
the compiler doesn't print any warnings/errors (though the generated code raises `AMX_ERR_INVSTATE` as function `func()` isn't implemented for the default state).

Maybe we should close this issue as invalid?

**Comment by Y-Less** [comment]
Yes, variables can have states as well.  And two variables in different states may share a memory location, which is quite nice.

**Comment by Daniel-Cortez** [comment]
Yes, you're right, I just haven't seen this feature used in the wild. I'll see what I can do about incorrect suggestions for state variables.

**Comment by Daniel-Cortez** [comment]
@pawn-lang Please close this (fixed in #431).
